packages/cli/src/commands/alias/rm.ts

```diff
@@ -59,7 +59,7 @@ export default async function rm(
 
   const removeStamp = stamp();
   if (!opts['--yes'] && !(await confirmAliasRemove(client, alias))) {
-    output.log('Aborted');
+    output.log('Canceled.');
```

Comment:
I think we should be consistent whether the period is present. I would argue that if it's only a single word, then omit the period.

Reply:
Thanks, just did this in the latest commit